examples/12-fluids/03-fluids_isosurface.py

```diff
@@ -101,9 +102,11 @@
     )
     min_pressure += step
 
-pl.show_figure(show_axes=True, cpos=c_pos_iso, return_cpos=True)
+pl.show_figure(show_axes=True, cpos=c_pos_iso)
 
 ###############################################################################
+# Important note
+# ------------------------------
 # For elemental or face variables we should have done an averaging on node
```

Comment:
It's better to state the reasons than the workarounds:

```
# Isosurfaces computation through the `mesh_cut` operator are only supported for Nodal Fields.
# Thus, for Elemental variables an averaging operation to the Nodes needs to be performed before
# running the `mesh_cut` operator. This can be done by chaining the `elemental_to_nodal` operator
# output with the `mesh_cut` operator input.
```